src/pages/IndexPage/components/IndexTimetable/index.tsx

```diff
@@ -15,12 +15,14 @@ import styles from './IndexTimetable.module.scss';
 function CurrentSemesterTimetable() {
   const semester = useSemester();
   const token = useTokenState();
-  const { data: timetableFrameList } = useTimetableFrameList(token, semester);
   const [currentFrameIndex, setCurrentFrameIndex] = React.useState<number>(0);
+  const { data: timetableFrameList } = useTimetableFrameList(token, semester);
 
   useEffect(() => {
     if (timetableFrameList) {
-      const mainFrame = timetableFrameList.find((frame) => frame.is_main === true);
+      const mainFrame = timetableFrameList.find(
+        (frame) => frame.is_main === true,
+      );
```

Comment:
```suggestion
      const mainFrame = timetableFrameList.find((frame) => frame.is_main);
```
만약 이 부분을 간단하게 표현한다면 이런 식으로도 할 수 있을 것 같네용